### ISSUE TITLE
ERROR: No mapping for the Unicode character exists in the target multi-byte code page

### ISSUE THREAD

**Issue by digao-dalpiaz** [digao-dalpiaz/Updater#2]
When running a definition, if some file name contains Unicode characters, this error occurs, and the process is interrupted.

![unicode](https://user-images.githubusercontent.com/46692097/136718025-38c28191-e0ce-455c-9883-26b47287aa78.png)

**Comment by digao-dalpiaz** [comment]
https://stackoverflow.com/questions/35708827/what-could-cause-no-mapping-for-the-unicode-character-exists-in-the-target-mult

It's a bug in Delphi.

Disabling Log File in Updater should avoid this bug.

**Comment by sprudi** [comment]
You can avoid this with disabling Logging or set encoding to ANSII in UEngine.pas
--> TFile.AppendAllText(pubLogFile, DateTimeToStr(Now)+' - '+LogFilePrefix+Text + #13#10, **TEncoding.ANSI**);

**Comment by digao-dalpiaz** [comment]
> You can avoid this with disabling Logging or set encoding to ANSII in UEngine.pas --> TFile.AppendAllText(pubLogFile, DateTimeToStr(Now)+' - '+LogFilePrefix+Text + #13#10, **TEncoding.ANSI**);

Right, but the text being sent to the log file contains **unicode** characters. So, in this case, it would not be correct for me to include a line in the file forcing **ANSI** encoding, right?
